packages/client/src/components/ToolsGrid/ToolsGrid.component.js

```diff
@@ -25,6 +26,12 @@ export const ToolsGrid = () => {
 
   const sortedTools = useMemo(() => {
     let result = tools;
+    if (searchBarText) {
```

Comment:
since tools is an array of objects, here is an exercise for you 😁 do you know how to make this method run faster? (hint: use map.has method) (this is just a nit comment. i.e, no changes needed)

Reply:
Thanks for your review. What I understand that map.has method returns a boolean value based on if an element with specified key exists in a map or not.